### PR TITLE
GH: auto label conflict PR

### PR DESCRIPTION
**Pull request by ddevsr**
**Description**
I think this help

**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [ ] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

File: .github/label-conflict.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+          # To make sure the merge commit exactly matches the branch
+          detect_merge_changes: true # or true to handle as conflicts
+          # By default a comment will be left, adding `conflict_comment: ''` will disable comments
+          # The optional `${author}` will be replaced with the username of the pull request
+          conflict_comment: |
+            :wave: Hi, @${author},
+            We detected conflicts against the base branch :speak_no_evil:
+            You'll want to sync :arrows_counterclockwise: your branch with upstream!
+
+            Ref: https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/workflow.md#pushing-your-branch
```
Comment (kenjis) [inline .github/label-conflict.yml:40]: You mean https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/workflow.md#updating-your-branch ?

File: .github/label-conflict.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+name: Auto Label Conflicts
+on:
+  push:
+    branches:
+      - 'develop'
+      - '4.*'
```
Comment (kenjis) [inline .github/label-conflict.yml:6]: Should be removed?

File: .github/label-conflict.yml (hunk 1 above)
Comment (ddevsr) [reply .github/label-conflict.yml:40]: Ya, almost similiar. When i forget, i usually read on https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/workflow.md#pushing-your-branch

File: .github/label-conflict.yml (hunk 2 above)
Comment (ddevsr) [reply .github/label-conflict.yml:6]: Just PR, no needed push trigger. I will change

**Review by kenjis — APPROVED** [review]
This would help us!

File: .github/label-conflict.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  issues: write
+  pull-requests: write
+
+jobs:
+  auto-label:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: prince-chrismc/label-merge-conflicts-action@v3
+        with:
+          conflict_label_name: "stale"
```
Comment (paulbalandan) [inline .github/label-conflict.yml:23]: Can be single quotes.

File: .github/label-conflict.yml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+          # --- Optional Inputs ---
+          # To make sure the merge commit exactly matches the branch
+          detect_merge_changes: true # or true to handle as conflicts
+          # By default a comment will be left, adding `conflict_comment: ''` will disable comments
+          # The optional `${author}` will be replaced with the username of the pull request
+          conflict_comment: |
+            :wave: Hi, @${author},
+            We detected conflicts against the base branch :speak_no_evil:
+            You'll want to sync :arrows_counterclockwise: your branch with upstream!
```
Comment (paulbalandan) [inline .github/label-conflict.yml:34]:
```suggestion
            :wave: Hi, @${author},

            We detected conflicts in your PR against the base branch :speak_no_evil:
            You may want to sync :arrows_counterclockwise: your branch with upstream!
```

**Review by kenjis — APPROVED** [review]

**Comment by kenjis** [comment]
@ddevsr Thank you!

**Comment by kenjis** [comment]
Sorry, this does not work.
We need to rework.
1. the place of the file is wrong.
2. if the PR is not updated, the check does not run.

**Comment by ddevsr** [comment]
The file must in `workflows`?

**Comment by kenjis** [comment]
Yes.
See https://docs.github.com/en/actions/learn-github-actions/understanding-github-actions#create-an-example-workflow

**Comment by ddevsr** [comment]
@kenjis I make PR to moving #7928

**Comment by kenjis** [comment]
@ddevsr Thank you!

**Pull request merged**